app_metro_status/models.py

```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+from django.utils.translation import ugettext as _
+from django.db import models
+from django.utils import timezone
+from datetime import timedelta
+import json
+import requests
+
+from boites.models import Boite, App
+from app_metro_status import settings
+
+
+def maybe_json(json_data):
+    try:
+        return json.loads(json_data)
+    except ValueError:
+        return
+
+
+class AppMetroStatus(App):
+    failures = models.TextField(_(u'Pannes'))
+
+    def get_app_dictionary(self):
+        if not self.enabled:
+            return
+
+        # we want to update every VALUES_UPDATE_INTERVAL minutes
+        if timezone.now() <= self.last_activity + timedelta(minutes=settings.VALUES_UPDATE_INTERVAL):
+            return maybe_json(self.failures)
+
+        response = requests.get(settings.STAR_API_URL)
```

Comment:
requests can use a "params" argument (a dict) to build its query string. 

```python
STAR_API_URL = "{}search?dataset=tco-metro-lignes-etat-tr&facet=nomcourt&apikey={}".format(STAR_API_BASE_URL, STAR_API_KEY)
requests.get(settings.STAR_API_URL)
```

can be changed to 

```python
STAR_API_URL = "{}search".format(STAR_API_BASE_URL)

params = {
    'dataset': 'tco-metro-lignes-etat-tr',
    'facet': 'nomcourt',
    'apikey': STAR_API_KEY
}

response = requests.get(STAR_API_URL, params=params)
```

By the way, facet param is not needed in this case (don't know why they put it by default).